messages/deploy.json

```diff
@@ -29,7 +29,8 @@
     "validateDeployRequestId": "deploy request ID of the validated deployment to run a Quick Deploy",
     "soapDeploy": "deploy metadata with SOAP API instead of REST API",
     "predestructivechanges": "file path for a manifest (destructiveChangesPre.xml) of components to delete before the deploy",
-    "postdestructivechanges": "file path for a manifest (destructiveChangesPost.xml) of components to delete after the deploy"
+    "postdestructivechanges": "file path for a manifest (destructiveChangesPost.xml) of components to delete after the deploy",
+    "purgeOnDelete": "the deleted components in the destructiveChanges.xml manifest file aren't stored in the Recycle Bin. Instead, they become immediately eligible for deletion."
```

Comment:
@jshackell-sfdc I took this description right from [here](https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_deploy.htm#option_purgeOnDelete_desc) but let me know if it should be different

Reply:
I reworked it a bit, see below.